wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -203,7 +157,10 @@ class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
     }
 
     func redrawFont() {
-        updateFingerprint()
         updateLabel()
     }
+
+    func showOrHideMLS() {
+        mlsThumbprintLabel.heightAnchor.constraint(equalToConstant: 0).isActive = userClient?.mlsPublicKeys.ed25519 == nil
```

Comment:
suggestion: I would write this more "human readable". For example:
```
let hasMlsPublicKeys = userClient?.mlsPublicKeys.ed25519 != nil
mlsThumbprintLabelHeightConstraint.isActive = !hasMlsPublicKeys
```
and move `mlsThumbprintLabel.heightAnchor.constraint(equalToConstant: 0)` to the `mlsThumbprintLabelHeightConstraint` property.

Also the method name is very generic, maybe something like `showOrHideMLSPublicKeys()` or `showOrHideMLSInfo()`

Reply:
I see this more like a nitpick. I like to keep things concise. Is this really hard to understand?
 This could be name `showOrHideMLSInfo`

Reply:
Yes, it can be as nitpick. Let's ask more opinions @agisilaos or @netbe 
My suggestion was to improve and make the constraint change logic more readable.

Reply:
done